src/collection-browser.ts

```diff
@@ -263,9 +263,16 @@ export class CollectionBrowser
      *
      * We disable it during the automated scroll since we don't want to fetch pages for intervening cells the
      * user may never see.
+     *
+     * No need to fetch if placeholder is being displayed
      */
-    if (!model && !this.isScrollingToCell) {
+    if (!this.placeholderType && !model && !this.isScrollingToCell) {
       const pageNumber = Math.floor(offsetIndex / this.pageSize) + 1;
+      console.warn('****** FETCH PAGE tileModelAtCellIndex', {
```

Comment:
can you please our custom log() function to render logs messages for development purpose?

Reply:
yes I can remove these, actually once we get stable work on offshoot 👍